Hunting Queries/Microsoft 365 Defender/Lateral Movement/Account brute force.yaml

```diff
@@ -20,4 +20,4 @@ query: |
       FailedAccounts = makeset(iff(ActionType == "LogonFailed", Account, ""), 5),
       SuccessfulAccounts = makeset(iff(ActionType == "LogonSuccess", Account, ""), 5)
       by DeviceName, RemoteIP, RemoteIPType
-  | where Failed > 10 and Successful > 0 and FailedAccountsCount > 2 and SuccessfulAccountsCount == 1
```

Comment:
@sriley0975 This PR is for the new solution 'Authomize'. Can you please remove this change from this PR?

Reply:
The reason why this looks like a change is at some point I accidentally messed with the wrong file (has similar name to one of the Authomize files - Then after a few commits we realized this file was modified. I then updated the file with the same contents - this is exactly as it is in the original file now:
      FailedAccounts = makeset(iff(ActionType == "LogonFailed", Account, ""), 5),
      SuccessfulAccounts = makeset(iff(ActionType == "LogonSuccess", Account, ""), 5)
      by DeviceName, RemoteIP, RemoteIPType
  | where Failed > 10 and Successful > 0 and FailedAccountsCount > 2 and SuccessfulAccountsCount == 1
  
  I've asked @v-atulyadav for some assistance with this - as I only see two options - If I use revert changes then I get all the old data for my other files for that given commit. 
  
  So I need to work it out - Will give you an updated answer once I have a chat with a few people...

Reply:
Worked it out - We will close out this PR and submit again - I'll let @v-atulyadav know the new PR number via Teams. Thanks for your patience.